scheduler/src/clients/transformation-client.ts

```diff
@@ -15,7 +15,7 @@ export async function executeTransformation (transformationConfig: object): Prom
 
 export async function executeNotification (notificationConfig: NotificationConfig): Promise<void> {
   const response = await http.post(`notification/${notificationConfig.type}`, notificationConfig)
-  if (response.status !== 200) {
+  if (response.status !== 200 && response.status !== 204) {
```

Comment:
Not sure why we had 204 there at the beginning. But both is fine here, even if we do a 204 again (No Content) which might fit better as response for notifications